pkg/config/install_requirements.go

```diff
@@ -473,6 +489,8 @@ type RequirementsConfig struct {
 	AutoUpdate AutoUpdateConfig `json:"autoUpdate,omitempty"`
 	// BootConfigURL contains the url to which the dev environment is associated with
 	BootConfigURL string `json:"bootConfigURL,omitempty"`
+	// BuildPackConfig contains custom build pack settings
+	BuildPack BuildPackConfig `json:"buildPack,omitempty"`
```

Comment:
what's about making this 'buildPacks' opposed to 'buildPack' - making it clearer that this is the configuration for build packs in general and not for a given one.

Reply:
I spoke to @dgozalo about this and we both thought `buildPack` made more sense as it can only hold a single thing in this current time period.

Reply:
So the plural name can be misleading, but in the future it could make more sense if we start supporting multiple build packs in the future.

Reply:
The way I saw it was that we have configuration for build packs in general under this node. ATM, that would only be the build pack library location. So for me, "build pack" is in fact a bit misleading, since this is not a single build pack thing, but an overarching setting for build packs. But let's go for it if you think so.

Reply:
Agreed and changed